R/0temp_fileio_optional.R

```diff
@@ -0,0 +1 @@
+.spc_io_postprocess_optional <- hyperSpec::.spc_io_postprocess_optional
```

Comment:
@sangttruong Why is the mechanism of copying a function was preferred over exporting?

Reply:
Addressed in #3